### PR TITLE
JdsSelect: implement JdsOptions

### PR DESCRIPTION
**Pull request by adrianpdm**
### Task Description
[Integrasi JdsSelect dan JdsOptions](https://trello.com/c/RdWQq4RU/101-integrasi-jdsselect-jdsoptions)

### Changes
- Implement JdsOptions as JdsSelect dropdown/popover element
- Cleanup options-related SCSS within JdsSelect
- Rework JdsSelect keyboard navigation
- Add Storybook stories for JdsSelect variant

### Note
JdsSelect navigation currently assume JdsOptions popup will always appear on the bottom of input element.

### Preview
image

### CODE REVIEW

**Review by Ibwedagama — APPROVED** [review]

File: src/components/index.js [hunk 1]
```diff
@@ -12,9 +12,12 @@ export * from './JdsTextArea'
 export * from './JdsToggle'
 export * from './JdsSpinner'
 export * from './JdsIcon'
+<<<<<<< HEAD
+=======
 export * from './JdsOptions'
 export * from './JdsAccordion'
 export * from './JdsBadge'
+>>>>>>> f4474e526ddec64311676ce1602d83cd3070937c
```
[the remark this answers is not in the document]
Comment (adrianpdm) [reply src/components/index.js]: thank you, fixed in [this commit](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/57/commits/0cd512f93649ec1ec961f19fff550b5018d0b34c)

File: src/components/JdsSelect/Select.vue [hunk 2]
```diff
@@ -129,16 +125,49 @@ export default {
...
+      default: "value",
     },
     /**
      * Required if options consist of objects.
      * Defined option label.
      */
     labelKey: {
       type: String,
-      default: 'label',
+      default: "label",
```
Comment (yoslie) [inline src/components/JdsSelect/Select.vue:136]: **minor**
I checked on another component, there are still some use of single quote in this props default initiation section.
maybe need to be discussed in the team, whether to use single or double quotes for consistency of code.

File: src/components/index.js (hunk 1 above)
Comment (yoslie) [inline src/components/index.js]: **major**
merging code

Comment (yoslie) [reply src/components/index.js]: sorry, during the review Its already solved, just ignore this comment

**Review by Ibwedagama — COMMENTED** [review]
minor:

Screenshot from 2021-07-22 12-24-05

Based on the Design Metrics for `Options` Component, it appears that the color of the label should be green when it's hovered (and also move to the right a bit). However, on the Storybook's component preview, the color of the label doesn't change.

File: src/components/JdsSelect/Select.vue (hunk 2 above)
Comment (adrianpdm) [reply src/components/JdsSelect/Select.vue:136]: This will be fixed on another Pull Request, along with ESLint setup.

**Comment by adrianpdm** [comment]
> minor:
> 
> Screenshot from 2021-07-22 12-24-05
> 
> Based on the Design Metrics for `Options` Component, it appears that the color of the label should be green when it's hovered (and also move to the right a bit). However, on the Storybook's component preview, the color of the label doesn't change.

Fixed on [this commit](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/57/commits/2c77f47a875843970046e79eb98a548af671cb03).

**Review by yoslie — APPROVED** [review]

**Review by Arif9878 — APPROVED** [review]

**Review by maulanayuseph — APPROVED** [review]

**Review by adzharamrullah — APPROVED** [review]

**Review by maruf12 — APPROVED** [review]

**Pull request merged**